app/src/components/layout/Navbar/Navbar.tsx

```diff
@@ -80,7 +79,11 @@ export const Navbar: React.FC = () => {
           type="button"
           onClick={() => setIsOpened(!isOpened)}
           color="transparent">
-          {isOpened ? <CloseMenuIcon /> : <OpenMenuIcon />}
+          {isOpened ? (
```

Comment:
Again, if there's only one prop difference between the two, then you could do `<Icon icon={isOpened ? 'CloseMenuIcon' : 'OpenMenuIcon'} height={30} width={35} />`

Reply:
+1